json/json_test.go

```diff
@@ -499,7 +499,7 @@ func BenchmarkUnmarshal(b *testing.B) {
 				x = duration(d)
 			}
 
-			j, _ := json.Marshal(x)
+			j, _ := Marshal(x)
```

Comment:
I think this intended to use the stdlib json package to marshal, and then this package to unmarshal. Below `unmarshal` is (by default) a pointer to this package's Unmarshal.